xarray/core/duck_array_ops.py

```diff
@@ -149,6 +150,28 @@ def trapz(y, x, axis):
 )
 
 
+def astype(data, **kwargs):
+    try:
+        import sparse
+    except ImportError:
+        sparse = None
+
+    if (
+        sparse is not None
+        and isinstance(data, sparse._coo.core.COO)
```

Comment:
could we use `.pycompat.sparse_array_type` here?

Reply:
Good idea! Latest commit uses `sparse_array_type`